Small fix for status

UlysseVautier opened the pull request:
If this is not done, a KeyError happens when launching the command `pifi status`.

UlysseVautier committed:
Small fix for status

johnflux commented on pifi/pifi.py:
```diff
@@ -71,8 +71,9 @@ def status(nm=nm):
         current_connection = ApModeDevice.GetAppliedConnection(0)
-        if current_connection[0]['802-11-wireless']['mode'] == "ap":
-            print("Device is currently acting as an Access Point")
+        if 'mode' in current_connection[0]['802-11-wireless']:
```
This doesn't look right because if you have a 'mode' key but it isn't equal to "ap" then the else condition isn't run

UlysseVautier replied:
Oh yes you are right. I assumed that 'mode' was appearing only when 'ap' was available. I have changed this so that 'ap' mode is checked when 'mode' is in current_connection AND 'mode'=='ap'. Considering that python is safe in the order of the conditionals.

UlysseVautier committed:
Fix status issue with only mode 'ap'

The check for 'ap' is done both whether mode is in current_connection and if mode == 'ap'.

rohbotics commented:
Sorry for the delay in reviewing. This looks good.

Thanks for your contribution! I never actually expected any on pifi :D
